pbrd/pbr_zebra.c

```diff
@@ -3,6 +3,9 @@
  * Zebra connect code.
  * Copyright (C) 2018 Cumulus Networks, Inc.
  *               Donald Sharp
+ * Portions:
+ *		Copyright (c) 2021 The MITRE Corporation. All Rights Reserved.
```

Comment:
Can we drop "All Rights Reserved" and leave just the Copyright + the company name? And the below Approved...